src/components/List.js

```diff
@@ -7,7 +7,7 @@ import { differenceInDays, addDays } from 'date-fns';
 import './List.css';
 
 const List = ({ token }) => {
-  const { docs, errorMessage, deleteDoc } = useFirestore(token);
+  const { docs, errorMessage, deleteDoc, loading } = useFirestore(token);
```

Comment:
Nice, a simple `loading` attribute from our data library is a great way to check this!